hack/addons/kustomize/nfd/kustomization.yaml.tmpl

```diff
@@ -18,7 +18,7 @@ helmCharts:
   valuesFile: helm-values.yaml
   valuesInline:
     image:
-      tag: "${NODE_FEATURE_DISCOVERY_VERSION}-minimal"
+      tag: "v${NODE_FEATURE_DISCOVERY_VERSION}-minimal"
```

Comment:
Wait was the NFD Deployment never coming up because of a wrong image tag?

Reply:
NFD was definitely working, e2e test validates this

Reply:
this had something to do with mindthegap not being able to find the tag